test/e2e/framework/pv_util.go

```diff
@@ -295,53 +354,72 @@ func CreatePVsPVCs(numpvs, numpvcs int, c clientset.Interface, ns string, pvConf
 
 	// create pvs and pvcs
 	for i = 0; i < pvsToCreate; i++ {
-		pv, pvc = CreatePVPVC(c, pvConfig, pvcConfig, ns, false)
+		if pv, pvc, err = CreatePVPVC(c, pvConfig, pvcConfig, ns, false); err != nil {
+			return pvMap, pvcMap, err
```

Comment:
I reverted my change to clean up in CreatePVsPVCs() and made a minor change to make the scope of the two maps clearer in _persistent_volumes.go_